lib/install/web.rb

```diff
@@ -1,4 +1,27 @@
+def node_version
+  ENV["NODE_VERSION"] || `node --version`[/\d+\.\d+\.\d+/]
+end
+
+def node_not_installed?
+  !node_version.present?
+end
+
+def node_version_unsupported?
+  node_version < "20.0.0"
+end
```

Comment:
I needed to re-declare these since this template will be treated as a static file when invoked. I can't import Suspenders.